client/src/components/ErrorBox.js

```diff
@@ -0,0 +1,21 @@
+import React, { Component } from "react";
+
+const errorStyle = {
+    float: "right",
```

Comment:
I think it'd be better to make this error show up closer to where the user is typing so they're more likely to notice it

Reply:
Although possible, I do not know how I would make it look good (i.e. this is the best looking implementation that I could think of.)

Reply:
To do that I'm pretty sure you would make three columns in the chat page: an empty one on the left, the name in the middle, and the button on the right